front/src/common/redux/actions/types.js

```diff
@@ -1,2 +1,11 @@
 export const CALC = 'CALC';
 export const APP_LOAD = 'APP_LOAD';
+
+export const LOGIN_USER = 'LOGIN_USER';
+export const LOGIN_USER_SUCCESS = 'LOGIN_USER_SUCCESS';
+export const LOGIN_USER_ERROR = 'LOGIN_USER_ERROR';
+
+export const REGISTER_STUDENT = 'REGISTER_STUDENT';
+export const REGISTRATION_STUDENT = 'REGISTRATION_STUDENT';
```

Comment:
For successful cases use the word `SUCCESS` in the end. It is much easier to understand the purpose of particular action